blog.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="./css/blog.css">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>MY BRAND</title>
+</head>
+<body>
+<nav class="fixed-nav-bar" id="navbar">
+    <div class="logo">
+        <h3><a href="index.html">mr chris</a></h3>
+    </div>
+    <ul class="nav-links">
+        <li><a href="index.html">Home</a></li>
```

Comment:
@Mucyo-Aime-Christian, since your home page contains many sections, please use # to link to a given class instead of linking to a new page.